utils/build/docker/python/flask/app.py

```diff
@@ -1099,20 +1099,10 @@ def view_iast_ssrf_insecure():
 
 @app.route("/iast/ssrf/test_secure", methods=["POST"])
 def view_iast_ssrf_secure():
-    from urllib.parse import urlparse
-
     import requests
 
-    url = flask_request.form["url"]
```

Comment:
For my information, it does not change IAST behaviour if the url has not been readed from `form` ?

Reply:
Reading it from `form` makes sure it's tainted.